steps.sh

```diff
@@ -0,0 +1,37 @@
+steps="${step_context}"
+job=$1
+repo_url="https://github.com/$2"
+run_url="https://github.com/$2/actions/runs/$3"
+results=""
+if [ -z "$steps" ]
+then
+  exit 0
+else
+  for k in $(jq -c 'keys[]' <<< "$steps"); do
+    outcome="$(jq -c ".${k}.outcome" <<< "$steps")"
+    if [[ $outcome != \"success\" ]]
+    then
+      if [ -z "$results" ]
+      then
+        repo_url="$(sed 's/&/\&amp;/g; s/</\&lt;/g; s/>/\&gt;/g; s/"/\&quot;/g; s/'"'"'/\&#39;/g' <<< "$repo_url")"
```

Comment:
It would be nice not to repeat this parameter all the time.

Reply:
Agreed, done